meteor/server/api/integration/rabbitMQ.ts

```diff
@@ -211,8 +211,12 @@ class ChannelManager extends Manager<AMQP.ConfirmChannel> {
 	triggerHandleOutgoingQueue() {
 		if (!this.handleOutgoingQueueTimeout) {
 			this.handleOutgoingQueueTimeout = setTimeout(() => {
-				this.handleOutgoingQueueTimeout = null
-				this.handleOutgoingQueue()
+				try {
+					this.handleOutgoingQueueTimeout = null
+					this.handleOutgoingQueue()
+				} catch (e) {
+					logger.error(`Unexpected error in AMQP triggerHandleOutgoingQueue`)
```

Comment:
I feel like this should call `this.triggerHandleOutgoingQueue()` to ensure that the queue keeps moving even when it does throw